### PR TITLE
Terraform readme

### PR DESCRIPTION
**Pull request by YakMan101**
Tiny change for reliability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,3 +47,6 @@ This project is focused on creating an **automated data analysis pipeline** to a
 #### 🗺️ Database Schema
 ![Database Schema](schema.png)
 
+## Infrastructure Setup
+
+Please follow the instructions laid out in `terraform/README.md`
```
Comment (mehrdadhalali) [inline README.md:52]: Lovely instruction

File: dashboard/upload_dashboard.sh [hunk 2]
```diff
@@ -17,8 +17,8 @@ ssh -i "$KEY_PATH" $EC2_USER@$EC2_HOST << EOF
     fi
     
     # Create a Python virtual environment and install dependencies
-    python3 -m venv venv
-    source venv/bin/activate
+    python3 -m venv .venv
```
Comment (Leon-JS02) [inline dashboard/upload_dashboard.sh:20]: Why `.venv` over `venv`?

**Review by mehrdadhalali — APPROVED** [review]

**Review by Leon-JS02 — APPROVED** [review]
Good readability changes.

File: dashboard/upload_dashboard.sh (hunk 2 above)
Comment (YakMan101) [reply dashboard/upload_dashboard.sh:20]: Got errors from EC2 when it was `venv` for some reason beyond my understanding.

File: README.md (hunk 1 above)
Comment (Ruyzambrano) [reply README.md:52]:
```suggestion
Please follow the instructions laid out in [the terraform directory](./terraform/README.md)
```

File: dashboard/upload_dashboard.sh (hunk 2 above)
Comment (Ruyzambrano) [reply dashboard/upload_dashboard.sh:20]: Using a `.` beforehand suggests that the file is a hidden file, which a `.venv` technically should be

File: terraform/README.md [hunk 3]
```diff
@@ -16,9 +16,11 @@ And the following services provisioned:
 ## 📂 Setup
 Follow these steps to deploy the infrastructure:
 
-1. Clone the repository to your local machine.
+1. Clone the repository to your local machine if you haven't already.
 
-2. Create a `terraform.tfvars` file in this directory with your AWS credentials and infrastructure details:
+2. Perform the short and long term ETL setup by reading `../short_term/README.md` and `../long_term/README.md`
```
Comment (Ruyzambrano) [inline terraform/README.md:21]: What do you mean by reading here? Do you mean following the steps outlined in the [short term](../short_term/README.md) and [long term](../long_term/README.md) documentation?

File: terraform/README.md [hunk 4]
```diff
@@ -42,29 +44,38 @@ ECR_SHORT_TERM_REPO_NAME  =  "name-of-repo-to-store-short-term-etl-image"
...
 
-This will create the necessary resources for the ETL pipeline, including:
-- Lambda for short-term ETL
-- ECS task definition for the long-term ETL
-- EventBridge schedules to trigger the ETL pipelines
-- IAM roles and policies for Lambda, ECS, and EventBridge
-- EC2 to host the dashboard
+7. Restrict access to the generated pemkey
+```bash
+chmod 400 c13-rvbyaulf-lmnh-key-pair.pem
```
Comment (Ruyzambrano) [inline terraform/README.md:64]: Should you remove the name of your pemkey here?

File: terraform/README.md [hunk 5]
```diff
@@ -42,29 +44,38 @@ ECR_SHORT_TERM_REPO_NAME  =  "name-of-repo-to-store-short-term-etl-image"
...
+```
+
+8. Follow the instructions in `../dashboard/README.md` to setup and run the dashboard.
```
Comment (Ruyzambrano) [inline terraform/README.md:67]:
```suggestion
8. Follow the instructions in [the dashboard README](../dashboard/README.md) to setup and run the dashboard.
```

**Pull request merged**